src/nitclk/system_tests/tests_system_nitclk.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+
+import datetime
+import fasteners
+import niscope
+import nitclk
+import os
+import pytest
+import tempfile
+
+
+@pytest.fixture(scope='function')
+def session_5164():
+    with niscope.Session('', False, False, 'Simulate=1, DriverSetup=Model:5164;BoardType:PXIe') as simulated_session:
+        yield simulated_session
+
+
+@pytest.fixture(scope='function')
+def session_multiple_sessions():
```

Comment:
Consider using consistent names to the fixtures that don't reference the specific model.
Like `single_niscope_session` and `multiple_niscope_sessions`

Reply:
Changed